framework/src/wifi_manager/wifi_manager_state.c

```diff
@@ -320,6 +321,55 @@ wifi_manager_result_e _wifimgr_scan(wifi_manager_scan_config_s *config)
 	return WIFI_MANAGER_SUCCESS;
 }
 
+static wifi_manager_result_e _wifimgr_scan_multi_aps(wifi_manager_scan_multi_configs_s *configs)
+{
+	if (!configs) {
+		WIFIMGR_CHECK_UTILRESULT(wifi_utils_scan_multi_aps(NULL), TAG,
+								 "request scan multi aps to wifi utils is fail");
+		return WIFI_MANAGER_SUCCESS;
```

Comment:
Debug message says `is fail` but return value is `SUCCESS`. Is it correct?

Reply:
> Debug message says `is fail` but return value is `SUCCESS`. Is it correct?

Yes.
That message is for when the `wifi_utils_scan_multi_aps` execution fails. You can see `WIFIMGR_CHECK_UTILRESULT` macro.